### PR TITLE
Align CBOR production/consumption rules with rest of specification.

### PR DESCRIPTION
**Pull request by msporny**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/476.html" title="Last updated on Dec 13, 2020, 8:33 PM UTC (4d5c073)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/476/05d75f3...4d5c073.html" title="Last updated on Dec 13, 2020, 8:33 PM UTC (4d5c073)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2804,38 +2804,132 @@ <h2>CBOR</h2>
...
+A <a>DID document</a> MUST be a single
+<a data-cite="RFC7049##section-2.1">CBOR Map</a> conforming to [[RFC7049]].
+All top-level properties of the <a>DID document</a> MUST be represented by
+using the property name as the name of the key of the CBOR Map. The values
+of properties of the data model described in Section <a href="#data-model"></a>,
+including all extensions, are encoded in CBOR [[RFC7049]] by mapping property
+values to CBOR types as follows:
+        </p>
+
+        <table class="simple" id="vanilla-cbor-representation-production">
```
Comment (jonnycrunch) [inline index.html:2830]: Vanilla CBOR might not be the best characterization.

**Review by brentzundel — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (OR13) [inline index.html:2830]:
```suggestion
        <table class="simple" id="cbor-representation-production">
```

**Review by OR13 — CHANGES_REQUESTED** [review]
prefer for us to finish describing "cbor" well before attempting to add subclasses of cbor to the spec

File: index.html (hunk 1 above)
Comment (jonnycrunch) [reply index.html:2830]: Agree with @OR13 !

**Comment by msporny** [comment]
> prefer for us to finish describing "cbor" well before attempting to add subclasses of cbor to the spec

Noting the "request changes" -- can I raise that in another PR? I assume this means you'd rather /not/ see the CBOR Extensions and DagCBOR stuff in the spec until we get the basics of CBOR hammered down, @OR13?

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**